### PR TITLE
docs(aio): upgrade to v5 and cli 1.5

### PR DESCRIPTION
**Pull request by Foxandxss**
No errors for v5, just a minor upgrade for styleguide with the latest in-mem

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "zone.js": "^0.8.16"
   },
   "devDependencies": {
-    "@angular/cli": "^1.3.0",
+    "@angular/cli": "1.3.0",
```
Comment (petebacondarwin) [inline aio/package.json]: Did you mean to make this change?

File: aio/tools/examples/shared/package.json [hunk 2]
```diff
@@ -13,19 +13,19 @@
...
-    "@angular/core": "~4.3.1",
-    "@angular/forms": "~4.3.1",
-    "@angular/http": "~4.3.1",
-    "@angular/platform-browser": "~4.3.1",
-    "@angular/platform-browser-dynamic": "~4.3.1",
-    "@angular/platform-server": "~4.3.1",
-    "@angular/router": "~4.3.1",
-    "@angular/upgrade": "~4.3.1",
-    "angular-in-memory-web-api": "~0.4.6",
+    "@angular/animations": "~5.0.0",
```
Comment (petebacondarwin) [inline aio/tools/examples/shared/package.json]: why `~` here but `^` in i18n package.json?

File: aio/package.json (hunk 1 above)
Comment (Foxandxss) [reply aio/package.json]: certainly not, reverting.

File: aio/yarn.lock [hunk 3]
```diff
@@ -22,7 +22,7 @@
   dependencies:
     tslib "^1.7.1"
 
-"@angular/cli@^1.3.0":
+"@angular/cli@1.3.0":
```
Comment (petebacondarwin) [inline aio/yarn.lock]: and this change?

File: aio/tools/examples/shared/package.json (hunk 2 above)
Comment (Foxandxss) [reply aio/tools/examples/shared/package.json]: this drivers the entire docs, packages shared across all examples. the i18n package json is the CLI package.json, different meanings. It won't install anything, just for zips and the scripts.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Can you revert the changes to the AIO package.json and yarn.lock.
Otherwise LGTM

**Review by petebacondarwin — APPROVED** [review]

**Pull request closed without merging**